### PR TITLE
Pipelines

### PR DESCRIPTION
**Pull request by TordJoranger**

### CODE REVIEW

File: tests/QueueReceiver.UnitTests/Infrastructure/Repositories/ProjectRepositoryTests.cs [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
+                new Project
+                {
+                    ProjectId = projectId,
+                    ParentProjectId = null,
+                    PlantId = plantA,
+                    IsVoided = false
+                }
+            };
+
+            var mockSet =  projects.AsQueryable().BuildMockDbSet<Project>();
```
Comment (llykkelarsen) [inline tests/QueueReceiver.UnitTests/Infrastructure/Repositories/ProjectRepositoryTests.cs]: Type specification is reduntant

File: src/QueueReceiver.Worker/QueueReceiver.Worker.csproj [hunk 2]
```diff
@@ -8,15 +8,9 @@
     <OutputType>Exe</OutputType>
     <StartupObject />
     <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
```
Comment (llykkelarsen) [inline src/QueueReceiver.Worker/QueueReceiver.Worker.csproj:12]: Does this need to be specified when true is default?

File: tests/QueueReceiver.UnitTests/Core/Services/PlantServiceTests.cs [hunk 3]
```diff
@@ -31,6 +31,7 @@ public async Task GetPlantIdTest()
             //Act
             var result = await _plantService.GetPlantId(existingPlantOid);
 
+            //Assert
             Assert.AreEqual(existingPlantId,result);
         }
     }
```
Comment (llykkelarsen) [inline tests/QueueReceiver.UnitTests/Core/Services/PlantServiceTests.cs:37]: Redundant parentheses behind TestClass ln 9 & TestMethod ln 21

Comment (llykkelarsen) [reply tests/QueueReceiver.UnitTests/Core/Services/PlantServiceTests.cs:37]: Also, redundant type specification on line 29. ( <string?> )

File: tests/QueueReceiver.UnitTests/Core/Services/PersonProjectServiceTests.cs [hunk 4]
```diff
@@ -38,15 +40,17 @@ public async Task GiveAccessToPlantTest()
...
+                .Returns(Task.FromResult(new List<Project> { new Project { PlantId = plantId, ProjectId = projectId } }));
 
             //Act
             await service.GiveProjectAccessToPlant(personId, plantId);
-            Assert.Fail();
+            personProjectRepository.Verify(ppr => ppr.AddAsync(projectId,personId), Times.Once);
+            personProjectRepository.Verify(ppr => ppr.SaveChangesAsync(), Times.Once);
         }
 
         [TestMethod()]
```
Comment (llykkelarsen) [inline tests/QueueReceiver.UnitTests/Core/Services/PersonProjectServiceTests.cs]: Redundant parentheses here & TestClass, TestMethod

File: tests/QueueReceiver.UnitTests/Core/Services/PersonProjectServiceTests.cs [hunk 5]
```diff
@@ -2,7 +2,9 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
```
Comment (llykkelarsen) [inline tests/QueueReceiver.UnitTests/Core/Services/PersonProjectServiceTests.cs]: Unused using statement on ln 1

File: src/QueueReceiver.Core/Services/PersonService.cs [hunk 6]
```diff
@@ -58,12 +58,12 @@ public async Task<Person> FindOrCreate(string userOid)
...
             await _personRepository.SaveChangesAsync();
+            }
             return person;
         }
 
-        private async Task<Person> FindUseByEmailOrUserName(AdPerson adPerson)
+        private async Task<Person> FindUserByEmailOrUserName(AdPerson adPerson)
         {
             var person = await _personRepository.FindByUsername(adPerson.Username)
                  ?? await _personRepository.FindByUserEmail(adPerson.Email);
```
Comment (llykkelarsen) [inline src/QueueReceiver.Core/Services/PersonService.cs]: fix weird indentation

File: src/QueueReceiver.Worker/QueueReceiver.Worker.csproj (hunk 2 above)
Comment (TordJoranger) [reply src/QueueReceiver.Worker/QueueReceiver.Worker.csproj:12]: yep, doesn't pack unless its added. Source: 2 hours of debugging

**Pull request merged**